model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -279,6 +279,28 @@ object Workspace {
                        attributes: AttributeMap,
                        state: WorkspaceState
   ) =
+    buildWorkspace(namespace,
+                   name,
+                   workspaceId,
+                   createdDate,
+                   lastModified,
+                   createdBy,
+                   attributes,
+                   state,
+                   WorkspaceType.McWorkspace
+    )
+  def buildWorkspace(namespace: String,
+                     name: String,
+                     workspaceId: String,
+                     createdDate: DateTime,
+                     lastModified: DateTime,
+                     createdBy: String,
+                     attributes: AttributeMap,
+                     state: WorkspaceState,
+                     workspaceType: WorkspaceType
+  ) = {
+    val googleProjectId =
+      if (workspaceType == WorkspaceType.RawlsWorkspace) GoogleProjectId("google-id") else GoogleProjectId("")
```

Comment:
I thought this was fishy, until I realized this was the test constructor. 
I still think its fishy, but only for existing reasons.